backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -135,21 +138,42 @@ public void testUpdateToken_success() throws Exception {
         // Assert.
         assertEquals("token map status is", inMemoryVirtualQueueRepository.getTokenMap().get(tokenId).getStatus(), updateTokenStatusResponse.getUpdatedToken().getStatus());
     }
+    
+    @Test
+    public void testUpdateShop_success() throws Exception {
+        // Arrange.
+        UUID shopId = addShopToRepository(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
+        addTokenListToShop(shopId);
+        UpdateShopStatusRequest updateShopStatusRequest = new UpdateShopStatusRequest(shopId, ShopStatus.DELETED);
 
-    private UUID addTokenToTokenMap() {
-        UUID tokenId = UUID.randomUUID();
-        UUID shopId = UUID.randomUUID();
-        Token token = new Token(tokenId, shopId, 1);
-        token.setStatus(Status.ACTIVE);
-        inMemoryVirtualQueueRepository.getTokenMap().put(tokenId, token);
-        return tokenId;
+        // Act.
+        UpdateShopStatusResponse updateShopStatusResponse = inMemoryVirtualQueueRepository.updateShop(updateShopStatusRequest);
+
+        // Assert.
+        assertEquals("Shop status is", inMemoryVirtualQueueRepository.getShopMap().get(shopId).getStatus(), updateShopStatusResponse.getShop().getStatus());
```

Comment:
Have the  value "ShopStatus.DELETED" here instead of reading from repo.

Reply:
Done.